airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -105,6 +110,16 @@
     )
     # [END howto_google_display_video_deletequery_report_operator]
 
+    # [START howto_google_display_video_upload_multiple_entity_read_files_to_big_query]
+    upload_erf_to_bq = GCSToBigQueryOperator(
+        task_id='upload_erf_to_bq',
+        bucket=BUCKET,
+        source_objects=ERF_SOURCE_OBJECT,
+        destination_project_dataset_table='airflow_test.gcs_to_bq_table',
```

Comment:
Can you parameterize this task by environment variables to isolate multiple test runs?

Reply:
Sure.